posthog/exceptions.py

```diff
@@ -31,7 +30,8 @@ def generate_exception_response(
     """
     Generates a friendly JSON error response in line with drf-exceptions-hog for endpoints not under DRF.
     """
-    statsd.incr(
-        f"posthog_cloud_raw_endpoint_exception", tags={"endpoint": endpoint, "code": code, "type": type, "attr": attr}
-    )
+
+    from posthog.internal_metrics import incr
```

Comment:
Usually inter-function imports are used to avoid circular imports ruining things. :)

Reply:
yes, sorry I wasn't clear in my comment. I assumed we're doing this because of circular imports, so my question was why did we opt to solve the circular imports problem in this file and all the others we use `internal_metrics` compared to doing inter-function imports inside `internal_metrics` files (I'd assume that would also work, correct me if I'm wrong)?

Reply:
It might, didn't think too deeply on it. :) I don't think it matters much, do you?

Reply:
Not really just for next time to consider, it seemed like there would be less inter-function imports that way and the users of the functions added wouldn't need to think about it, which is what I like to base that choice on.